rest/model/commit_queue.go

```diff
@@ -88,13 +91,45 @@ func ParseGitHubComment(comment string) GithubCommentCqData {
 		if index == 1 {
 			data = parseFirstLine(line)
 		} else if index == 2 {
-			data.MessageOverride = line
+			data.MessageOverride = wrapString(line, commitMessageLineLength)
 		}
 	}
 
 	return data
 }
 
+// wrapString manually splits a string into smaller chunks and appends them to a new string.
+func wrapString(str string, limit int) string {
+	var b strings.Builder
+	currentLine := ""
+	lines := strings.Split(str, "\n")
+	for i, line := range lines {
+		words := strings.Fields(line)
+		for _, word := range words {
+			lineLength := len(currentLine + word)
+			// Only factor in spaces if the current line is not empty.
+			if len(currentLine) > 0 {
+				lineLength += 1
+			}
+			if lineLength > limit {
+				b.WriteString(currentLine + "\n")
+				currentLine = ""
+			}
+			if currentLine == "" {
+				currentLine += word
+			} else {
+				currentLine += " " + word
+			}
+		}
+		b.WriteString(currentLine)
+		if i < len(lines)-1 {
+			b.WriteString("\n")
```

Comment:
Is the thinking here that we want to preserve newlines as they originally appeared in the commit message, even if we're breaking up individual long lines?

Reply:
Yes, the assumption is that if someone introduced a line break in their initial message that it was intentional.